pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/DownloadParticipantListRoute.java

```diff
@@ -0,0 +1,181 @@
+package org.broadinstitute.dsm.route;
+
+import javax.servlet.ServletOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import com.google.gson.Gson;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.util.CellRangeAddress;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.broadinstitute.dsm.model.ParticipantColumn;
+import org.broadinstitute.dsm.model.elastic.sort.Alias;
+import org.broadinstitute.dsm.model.excel.ParticipantRecord;
+import org.broadinstitute.dsm.model.filter.FilterFactory;
+import org.broadinstitute.dsm.model.filter.Filterable;
+import org.broadinstitute.dsm.model.participant.DownloadParticipantListPayload;
+import org.broadinstitute.dsm.model.participant.ParticipantWrapperDto;
+import org.broadinstitute.dsm.model.participant.ParticipantWrapperResult;
+import org.broadinstitute.dsm.security.RequestHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+
+public class DownloadParticipantListRoute extends RequestHandler {
+
+    private static final Logger logger = LoggerFactory.getLogger(DownloadParticipantListRoute.class);
+    private static final String FILE_DATE_FORMAT = "ddMMyyyy";
+
+    public DownloadParticipantListRoute() {
+    }
+
+    @Override
+    public Object processRequest(Request request, Response response, String userId) throws Exception {
+        DownloadParticipantListPayload payload = new Gson().fromJson(request.body(), DownloadParticipantListPayload.class);
```

Comment:
because of we faced strange behavior from `gson` when it converts all the numbers to `double` we decided to slowly eliminate it from the project and replace it with `jackson`, there is a special classes created for it called `ObjectMapperSingleton`, it is preferable to use it.